docs/running/wdl.rst

```diff
@@ -0,0 +1,139 @@
+WDL Support in Toil
```

Comment:
Other docs building error:
`checking consistency... /Users/jesse/Documents/genomics/toil/docs/running/wdl.rst:: WARNING: document isn't included in any toctree`

This means the only way to access the WDL docs is to manually navigate to toil/docs/_build/html/running/wdl.html

Reply:
Added to index.rst.